fyle_accounting_mappings/models.py

```diff
@@ -650,3 +650,37 @@ def create_or_update_category_mapping(
         )
 
         return category_mapping
+
+
+class ExpenseFields(models.Model):
+    """
+    Expense Fields
+    """
+
+    id = models.AutoField(primary_key=True)
+    attribute_type = models.CharField(null=True, max_length=255, help_text='Attribute Type')
+    source_field_id = models.IntegerField(null=True, max_length=255, help_text='Field ID')
+    workspace = models.ForeignKey(Workspace, on_delete=models.PROTECT, help_text='Reference to Workspace model')
+    is_enabled = models.BooleanField(default=False, help_text='Is the field Enabled')
+    created_at = models.DateTimeField(auto_now_add=True, help_text='Created at datetime')
+    updated_at = models.DateTimeField(auto_now=True, help_text='Updated at datetime')
+
+    class Meta:
+        db_table = 'expense_fields'
+
+
+    @staticmethod
+    def create_or_update_expense_fields(attribute: Dict, workspace_id):
```

Comment:
how about passing a list to this function and then looping over it?
No asking to do bulk query but the function contract can still be bulk, later we can just swap out loop with bulk query